### PR TITLE
867 content updates for the voluntary conversions workflow

### PR DESCRIPTION
**Pull request by titlescreen**
## Changes

Iterate content with things we learned from private beta and regional delivery officers

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Comment by titlescreen** [comment]
I did a thing to make it work locally then pushed. There must be a better way or I'm doing it wrong, so please let me know the best practice.

**Comment by titlescreen** [comment]
Thanks Joseph, that content looks good to me.

File: app/workflows/lists/conversion/voluntary/sections/ready-for-opening.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ tasks:
       the Baseline email. Indicate on the Baseline that the school is on track
       to open.
     actions:
-      - title: Confirm the school is on track to open on the target date
+      - title: Confirm the school is on track to open on the provisional date
```
Comment (lozette) [inline app/workflows/lists/conversion/voluntary/sections/ready-for-opening.yml:10]: The previous commit swaps these over. I'd drop the previous commit as I assume it's not needed.

File: Gemfile.lock [hunk 2]
```diff
@@ -413,6 +413,7 @@ PLATFORMS
   arm64-darwin-21
   x86_64-darwin-20
   x86_64-darwin-21
+  x86_64-darwin-22
```
Comment (lozette) [inline Gemfile.lock:416]: Can this be moved into its own commit with its own message!

**Review by lozette — CHANGES_REQUESTED** [review]
Can you drop one commit entirely eec587467d8213f7d3310fb2557f899402aa3bed as it is overwritten in a later commit.
